style.css

```diff
@@ -0,0 +1,89 @@
+@import 'https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap';
+
+* {
+  left: 0;
+  top: 0;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.main-header {
+  display: flex;
+  justify-content: right;
+  margin-top: 63px;
+  flex-wrap: wrap;
+}
+
+.menu {
+  margin-right: 24px;
+}
+
+.logo {
+  margin-right: auto;
+  margin-left: 24px;
+}
+
+.part1 {
+  background: url(images/background.jpeg);
+  background-position: cover;
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 120px;
+}
+
+.head-name {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  margin: 110px 24px 10px;
+}
+
+.head-para {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  margin-right: 24px;
+  margin-left: 24px;
+}
+
+.sub-head {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  margin-left: 24px;
+}
+
+.social-media {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: fit-content;
+  padding: -20px;
+}
+
```

Comment:
- [ ] As suggested by the previous reviewer please 🙏 make social icons vertically aligned with the let’s connect link just like in the [figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=gHYreHPQqfVOzLLb-0). As I can see you have tried attempting to remove the space using `padding negative` value however it seems to be not working. The problem must be the default `padding` of the `ul` tag. I suggest adding `padding: 0;` and `margin: 0;`to your `*` block so that all the default paddings and margins are removed from all the elements and you can  customise your elements to your desire 😄

Your design:

<img width="376" alt="Screenshot 2023-03-23 at 11 21 10 AM" src="https://user-images.githubusercontent.com/44798044/227122821-8de65eba-65e2-47ed-9f65-03ce3b76e355.png">

Figma design:


<img width="451" alt="Screenshot 2023-03-23 at 11 36 18 AM" src="https://user-images.githubusercontent.com/44798044/227122939-3d896d3c-d0b6-4a30-9acf-b3cc546ece75.png">